start.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+export PYTHONPATH=$((dirname $0))
```

Comment:
Currently it gives me "syntax error: invalid arithmetic operator", the fix is:
```suggestion
export PYTHONPATH=$(dirname $0)
```

Another solution, which ensures absolute paths:
```suggestion
export PYTHONPATH="$(dirname "$(readlink -f "${BASH_SOURCE[0]}" )" )"
```

Reply:
@Mehehehehe82  Sorry for taking so much time to answer your simple PR.
What do you prefer between the 2 suggestions?

Reply:
Suggestion 2